src/elm/Auth.elm

```diff
@@ -176,21 +179,45 @@ passphraseValidator =
         ]
 
 
-{-| Validates PIN field and related PIN confirmation field.
--}
-pinValidator : Validator ( ValidatedField, String ) PrivateKeyLogin
+validatePinNumber : String -> List Problem
+validatePinNumber pin =
+    let
+        hasCorrectLength =
+            String.length pin == 6
+
+        hasOnlyDigits =
+            String.all Char.isDigit pin
+    in
+    if not hasCorrectLength || not hasOnlyDigits then
+        [ InvalidEntry PinConfirmation "PIN must be 6 digits (TR)" ]
```

Comment:
I know this is just a WIP, but please make sure the errors are localised! 😉

Reply:
Sure! Those `(TR)` in texts are reminders for me to add translations.

Reply:
nice! keep the good work! 💪